pom.xml

```diff
@@ -78,7 +78,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.2.0-SNAPSHOT</cdap.version>
+    <cdap.version>4.3.0-SNAPSHOT</cdap.version>
```

Comment:
Merged develop (which is just merged with release/2.0) into this branch and updated cdap version to 4.3.0-SNAPSHOT